src/app/tic-tac-toe.component.ts

```diff
@@ -52,7 +52,7 @@ export class TicTacToeComponent {
     } else {
       return '';
     }
-  }
+  } // review
```

Comment:
A `switch` statement (instead of a nested `if-else`) could enhance this method's readability. 

Alternatively, instead of a mapping function, you could use a plain object that maps the enum's values to the appropriate strings.